docs/content/manual/pre-release/upgrade/test-node-drain-policy.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Test Node Drain Policy Setting
+---
+
+## With `node-drain-policy` is `block-if-contains-last-replica`
+
```

Comment:
I suggest adding a note about the following v1.5.x changes.


1. Starting from v1.5.x, it is not necessary to check for the presence of `longhorn-admission-webhook` and `longhorn-conversion-webhook`. Please refer to the Longhorn issue [#5590](https://github.com/longhorn/longhorn/issues/5590) for more details.
2. Starting from v1.5.x, observe that the `instance-manager` pod is NOT evicted due to the consolidation of the Instance Manager Engine and Replica functionalities. Please refer to the Longhorn issue [#5208](https://github.com/longhorn/longhorn/issues/5208) for more details.

Reply:
> Starting from v1.5.x, observe that the instance-manager pod is NOT evicted due to the consolidation of the Instance Manager Engine and Replica functionalities. Please refer to the Longhorn issue https://github.com/longhorn/longhorn/issues/5208 for more details.

@roger-ryao what does this mean? 

cc @c3y1huang

Reply:
> > Starting from v1.5.x, observe that the instance-manager pod is NOT evicted due to the consolidation of the Instance Manager Engine and Replica functionalities. Please refer to the Longhorn issue [longhorn/longhorn#5208](https://github.com/longhorn/longhorn/issues/5208) for more details.
> 
> @roger-ryao what does this mean?
> 
> cc @c3y1huang

Hi @innobead :
    The test case step [Observe that instance-manager-r is NOT evicted](https://github.com/longhorn/longhorn-tests/pull/1395/files#diff-c8a07a00c0ddc3e8635bf09586ca4a2a14c65193b99092918c7f937c965637d8R20) needs to be updated after the 1.5.x version, where `instance-manager-r` and `instance-manager-e` are consolidated into a single `instance-manager` pod. This clarification will ensure a clear understanding of the changes in the test case.
    
WDYT

Reply:
Thanks @roger-ryao I added the note below.

The instance-manager-r and instance-manager-e are combine into instance-manager. However, they should still be evictable when applicable. WDYT?